dev-scripts/webterminal-localhost.sh

```diff
@@ -0,0 +1,165 @@
+
+      --frontend-patch: patches frontend to unblock you to compile and run manually
+
+      --install: propagating this script into PATH.
+                 The symlink is propagated into
+                 \$(systemd-path user-binaries) -> $(systemd-path user-binaries)
+      --uninstall: removing the installed symlink.
+
+      --help: get this info message
+
+      Usage:
+      execute \"./webterminal-localhost.sh --install\" to make it available from any place.
+
+      Then there are mainly two stories:
+
+      -- You are developing frontend --
+
+      1. Compile the patched backend:
+          ./webterminal-localhost.sh --backend
+      2. Compile the patched frontend and run bridge
+          ./webterminal-localhost.sh --frontend --run
+
+      2.* If you want to compile frontend by your own
+          ./webterminal-localhost.sh --frontend-patch (do not forget to exlude it before committing)
+        and then
+          ./webterminal-localhost.sh --run
+
+      -- You are developing backend --
+
+      1. Compile the patched frontend:
+          ./webterminal-localhost.sh --frontend
+      2. Compile the patched backend and run bridge
+          ./webterminal-localhost.sh --backend --run
+          or
+          ./webterminal-localhost.sh --emulate-in-cluster
+          ./examples/run-bridge.sh
+"
+}
+
+function install() {
+  rm -f $(systemd-path user-binaries)/webterminal-localhost.sh
+  ln -s $(dirname $(readlink -f $0))/webterminal-localhost.sh $(systemd-path user-binaries)/webterminal-localhost.sh 
+  echo "Symlink $(systemd-path user-binaries)/webterminal-localhost.sh is created
+Open a new terminal to get it propagated into path"
+}
+
+function uninstall() {
+  rm -f $(systemd-path user-binaries)/webterminal-localhost.sh
+}
+
+function emulate_in_cluster() {
+  mkdir -p /tmp/kube-in-cluster-emulation
+  SA_SECRET=$(oc get sa console -n openshift-console -o yaml | yq -r '.secrets[].name' | grep "console-token-.*")
+  oc get secret ${SA_SECRET} -n openshift-console -o json | jq -r '.data["ca.crt"]' | base64 -d > /tmp/kube-in-cluster-emulation/ca.crt
+  oc get secret ${SA_SECRET} -n openshift-console -o json | jq -r '.data["token"]'  | base64 -d > /tmp/kube-in-cluster-emulation/token
+
+  export KUBERNETES_SERVICE_PORT=6443                       
+  API=$(oc whoami --show-server)
+  API=${API##*://}
+  API=${API%%:*}
+  export KUBERNETES_SERVICE_HOST=${API}
+
+  if [[ ! -L "/var/run/secrets/kubernetes.io/serviceaccount"
+    || $( readlink -f /var/run/secrets/kubernetes.io/serviceaccount) != "/tmp/kube-in-cluster-emulation" ]]; then
+
+    echo "Require sudo access to create /var/run/secrets/kubernetes.io/serviceaccount/ directory
+      and create symlink to /tmp/kube-in-cluster-emulation"
+
+    sudo rm -f /var/run/secrets/kubernetes.io/serviceaccount
+    sudo mkdir -p /var/run/secrets/kubernetes.io/
+    sudo ln -s /tmp/kube-in-cluster-emulation /var/run/secrets/kubernetes.io/serviceaccount
```

Comment:
It may make sense to combine these into one command, otherwise on systems where the sudo timeout is 0 the user has to enter their password 3 times.
```suggestion
    sudo rm -f /var/run/secrets/kubernetes.io/serviceaccount &&\
         mkdir -p /var/run/secrets/kubernetes.io/ &&\
         ln -s /tmp/kube-in-cluster-emulation /var/run/secrets/kubernetes.io/serviceaccount
```